pkg/ansible/operator/operator.go

```diff
@@ -67,6 +79,24 @@ func Run(done chan error, mgr manager.Manager, f *flags.AnsibleOperatorFlags, cM
 	done <- mgr.Start(c)
 }
 
+func getMaxWorkers(gvk schema.GroupVersionKind, defvalue int) int {
+	envvar := formatEnvVar(gvk.Kind, gvk.Group)
+	maxWorkers, err := strconv.Atoi(os.Getenv(envvar))
+	if err != nil {
+		// we don't care why we couldn't parse it just use one.
+		// maybe we should log that we are defaulting to 1.
+		logf.Log.WithName("manager").V(0).Info("using default value for workers %d", defvalue)
```

Comment:
I’d say we should probably create a package variable with this and call it manager?

Reply:
I would generally agree but so far all of the uses of `Log.WithName(...)` use a string literal. I would either leave it alone or in a separate PR update all the uses of `Log.WithName(...)`. Personally I don't think it's a big deal to just have the string literal there.

Here are all the uses in the repo:
```
doc/user/logging.md:  log := logf.Log.WithName("cmd")
doc/user/logging.md:var log = logf.Log.WithName("controller_memcached")
example/memcached-operator/memcached_controller.go.tmpl:var log = logf.Log.WithName("controller_memcached")
pkg/ansible/controller/controller.go:var log = logf.Log.WithName("ansible-controller")
pkg/ansible/controller/reconcile.go:    logger := logf.Log.WithName("reconciler").WithValues(
pkg/ansible/controller/reconcile.go:    logger := logf.Log.WithName("markDone")
pkg/ansible/controller/status/types.go:var log = logf.Log.WithName("controller.status")
pkg/ansible/events/log_events.go:       logger := logf.Log.WithName("logging_event_handler").WithValues(
pkg/ansible/operator/operator.go:               logf.Log.WithName("manager").Error(err, "Failed to get watches")
pkg/ansible/operator/operator.go:               logf.Log.WithName("manager").V(0).Info("using default value for workers %d", defvalue)
pkg/ansible/proxy/kubeconfig/kubeconfig.go:var log = logf.Log.WithName("kubeconfig")
pkg/ansible/proxy/kubectl.go:var log = logf.Log.WithName("proxy")
pkg/ansible/proxy/requestfactory/requestinfo.go:var log = logf.Log.WithName("requestfactory")
pkg/ansible/run.go:var log = logf.Log.WithName("cmd")
pkg/ansible/runner/eventapi/eventapi.go:                logger:     logf.Log.WithName("eventapi").WithValues("job", ident),
pkg/ansible/runner/internal/inputdir/inputdir.go:var log = logf.Log.WithName("inputdir")
pkg/ansible/runner/runner.go:var log = logf.Log.WithName("runner")
pkg/helm/controller/controller.go:var log = logf.Log.WithName("helm.controller")
pkg/helm/run.go:var log = logf.Log.WithName("cmd")
pkg/k8sutil/k8sutil.go:var log = logf.Log.WithName("k8sutil")
pkg/leader/leader.go:var log = logf.Log.WithName("leader")
pkg/metrics/metrics.go:var log = logf.Log.WithName("metrics")
pkg/predicate/predicate.go:var log = logf.Log.WithName("predicate").WithName("eventFilters")
pkg/scaffold/ansible/main.go:           logf.Log.WithName("cmd").Error(err, "")
pkg/scaffold/cmd.go:var log = logf.Log.WithName("cmd")
pkg/scaffold/cmd_test.go:var log = logf.Log.WithName("cmd")
pkg/scaffold/controller_kind.go:var log = logf.Log.WithName("controller_{{ .Resource.LowerKind }}")
pkg/scaffold/controller_kind_test.go:var log = logf.Log.WithName("controller_appservice")
pkg/scaffold/helm/main.go:              logf.Log.WithName("cmd").Error(err, "")
test/test-framework/cmd/manager/main.go:var log = logf.Log.WithName("cmd")
test/test-framework/pkg/controller/memcached/memcached_controller.go:var log = logf.Log.WithName("controller_memcached")
vendor/sigs.k8s.io/controller-runtime/pkg/cache/cache.go:var log = logf.KBLog.WithName("object-cache")
vendor/sigs.k8s.io/controller-runtime/pkg/client/config/config.go:      log                   = logf.KBLog.WithName("client").WithName("config")
vendor/sigs.k8s.io/controller-runtime/pkg/client/fake/client.go:        log = logf.KBLog.WithName("fake-client")
vendor/sigs.k8s.io/controller-runtime/pkg/handler/enqueue.go:var enqueueLog = logf.KBLog.WithName("eventhandler").WithName("EnqueueRequestForObject")
vendor/sigs.k8s.io/controller-runtime/pkg/handler/enqueue_owner.go:var log = logf.KBLog.WithName("eventhandler").WithName("EnqueueRequestForOwner")
vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:var log = logf.KBLog.WithName("controller")
vendor/sigs.k8s.io/controller-runtime/pkg/manager/internal.go:var log = logf.KBLog.WithName("manager")
vendor/sigs.k8s.io/controller-runtime/pkg/predicate/predicate.go:var log = logf.KBLog.WithName("predicate").WithName("eventFilters")
vendor/sigs.k8s.io/controller-runtime/pkg/runtime/log/log.go:   KBLog = Log.WithName("kubebuilder")
vendor/sigs.k8s.io/controller-runtime/pkg/source/internal/eventsource.go:var log = logf.KBLog.WithName("source").WithName("EventHandler")
vendor/sigs.k8s.io/controller-runtime/pkg/source/source.go:var log = logf.KBLog.WithName("source")
vendor/sigs.k8s.io/controller-runtime/pkg/webhook/admission/doc.go:var log = logf.KBLog.WithName("admission")
```